terraform/environments/staging/relays.tf

```diff
@@ -100,6 +100,12 @@ module "relays" {
 
   application_name    = "relay"
   application_version = replace(var.image_tag, ".", "-")
+  application_environment_variables = [
```

Comment:
Rather than inject the variables here directly, the relay app module should be updated with a new telemetry boolean var that can be overridden here and injected as an env var there.

This will make it easier for anyone working with that module to know the configuration it expects (incl us).

See the observability log level as an example.

Reply:
I didn't want to do that because I didn't want (self-hosted) users copying the module to enable telemetry. I also don't know much about terraform though but it just felt like this is the right place to put this stuff.